index.html

```diff
@@ -6,6 +6,6 @@
     <title>Benvenuto a Potionfy</title>
 </head>
 <body>
-    
+    <h1>Benvenuti! Divertitevi!!!</h1>
```

Comment:
Puoi rimuovere la parola "Divertitevi!" e aggiungere la scritta "Benvenuti a Potionfy" in rosso ?

Reply:
bravo